code/obj/stool.dm

```diff
@@ -1118,6 +1231,84 @@
 		icon_state = "fpewR"
 		arm_icon_state = "arm-fpewR"
 
+	realign()
+		icon_state = "fpew"
+		arm_icon_state = "arm-fpew"
+		switch(src.dir)
+			if(NORTH)
+				for (var/obj/stool/chair/pew/P in get_step(src, EAST))
+					if(P.dir==src.dir)
+						icon_state = "fpewL"
+						arm_icon_state = "arm-fpewL"
+						P.alignadjacent(1)
+					break
+				for (var/obj/stool/chair/pew/P in get_step(src, WEST))
+					if(P.dir==src.dir)
+						icon_state = (icon_state=="fpewL") ? "fpewC" : "fpewR"
+						arm_icon_state = (arm_icon_state=="arm-fpewL") ? "null" : "arm-fpewR"
```

Comment:
see: gross copypasting comment

object oriented methodologies are supposed to prevent this grossness, please use inheritance properly